### ISSUE TITLE
Suggestion: ability to pause/stop copy process

### ISSUE THREAD

**Issue by warpdesign** [sindresorhus/cpy#53]
If I'm not mistaken there is no way to stop the copy process: it would be great if there was a way to do it.

**Comment by sindresorhus** [comment]
Yes, we could add `.stop()`, `.pause()`, `.resume()` methods to the promise.
